shared/naturalcrit/markdown.js

```diff
@@ -237,32 +237,45 @@ const superSubScripts = {
 const definitionLists = {
 	name  : 'definitionLists',
 	level : 'block',
-	start(src) { return src.match(/^.*?::.*/m)?.index; },  // Hint to Marked.js to stop and check for a match
+	start(src) { return src.match(/^.*?::.*\n\n/m)?.index; },  // Hint to Marked.js to stop and check for a match
 	tokenizer(src, tokens) {
-		const regex = /^([^\n]*?)::([^\n]*)(?:\n|$)/ym;
+		const regex = /^([^\n:]*?)::(.*)(?:\n|$)/ym;
 		let match;
-		let endIndex = 0;
-		const definitions = [];
+		const endIndex = src.match(`\n\n`)?.index + 2;
+		const allDefinitions = [];
+		let currentDefinition = [];
 		while (match = regex.exec(src)) {
-			definitions.push({
-				dt : this.lexer.inlineTokens(match[1].trim()),
-				dd : this.lexer.inlineTokens(match[2].trim())
-			});
-			endIndex = regex.lastIndex;
+			if(match[1].trim()?.length) {
+				if(currentDefinition?.dt?.length) {
+					allDefinitions.push(currentDefinition);
+					currentDefinition = [];
+				}
+				currentDefinition = {
+					dt : this.lexer.inlineTokens(match[1].trim()),
+					dd : []
+				};
+			}
+			currentDefinition.dd = currentDefinition.dd.concat(match[2].split('::').filter((item)=>item).map((s)=>this.lexer.inlineTokens(s.trim())));
```

Comment:
There is a crash here when typing the following line

```
Test
:: one
```
Can we get this to work? If we are allowing multi-line syntax I would like to be able to write each DD entry its own line.

Reply:
This should be resolved.